src/KubeOps/Operator/Builder/IOperatorBuilder.cs

```diff
@@ -57,5 +62,83 @@ IOperatorBuilder AddLivenessCheck<TLivenessCheck>(string? name = default)
         /// <param name="assembly">The assembly to add.</param>
         /// <returns>The builder for chaining.</returns>
         IOperatorBuilder AddResourceAssembly(Assembly assembly);
+
+        /// <summary>
+        /// <para>
+        /// Adds an entity to the operator to be considered for RBAC / CRD generation.
+        /// </para>
+        /// <para>
+        /// Only useful if a) the given type is not referenced by a controller, finalizer, or webhook
+        /// and b) the assembly containing the type is not already autypeoftomatically scanned.
```

Comment:
Yeah, I found out after doing all this that middle click pastes apparently. I guess I missed that one since IntelliSense didn't throw a syntax error at me.